src/config/watcher.rs

```diff
@@ -141,7 +141,6 @@ mod tests {
 
     #[tokio::test]
     async fn file_update() {
-        crate::test_util::trace_init();
```

Comment:
We probably shouldn't remove these from here.

Reply:
It looks like there are a lot of removals of the `crate::test_util::trace_init`, while this PR looks like it should add them. I'm confused, was the intent to remove or add them?

Reply:
Oops, looks like I removed this by accident. Will return back.

> It looks like there are a lot of removals of the `crate::test_util::trace_init`

In some places yes, I removed `trace_init` from setup function and add to functions itself.

> while this PR looks like it should add them.

Initially, I wanted to add `trace_init` to every function, but after @lukesteensen comment decide left everything. Changed title to more appropriate.